lib/charms/traefik_k8s/v1/ingress_per_unit.py

```diff
@@ -832,16 +831,16 @@ def urls(self) -> Dict[str, str]:
 
         May return an empty dict if the URLs aren't available yet.
         """
-        data = _type_convert_stored(self._stored.current_urls or {})  # type: ignore
```

Comment:
can we get rid of stored altogether?

Reply:
Currently it's used to compare current vs. previous for emitting the custom events.
Do you mean use peer data instead?